src/backend/sgx/data.rs

```diff
@@ -252,3 +252,32 @@ pub fn intel_crl() -> Datum {
         mesg: None,
     }
 }
+
+pub fn tcb_fmspc_cached() -> Datum {
+    const NAME: &str = "TCB & FMSPC cache";
+
+    if !Path::new(FMSPC_PATH).exists() {
+        return Datum {
+            name: NAME.to_string(),
+            pass: false,
+            info: Some("Missing FMSPC".into()),
+            mesg: Some("Run `enarx platform sgx cache-pck`".into()),
+        };
+    }
+
+    if !Path::new(TCB_PATH).exists() {
+        return Datum {
+            name: NAME.to_string(),
+            pass: false,
+            info: Some("Missing TCB report".into()),
+            mesg: Some("Run `enarx platform sgx cache-tcb`".into()),
```

Comment:
So, we now have three cache commands to call, which even need a specific order.
Would it make sense to combine all of these into one `update-cache` command?

Reply:
So it would be `enarx platform sgx update-cache` and `enarx platform snp cache-crl`? Shouldn't the commands be similar? Maybe @bstrie should weigh in on this.

Though both are updated about monthly, and maybe we could have a convenience alias: `enarx platform sgx update-cache` which would update TCB & CRLs?

Reply:
Issue for this: https://github.com/enarx/enarx/issues/2486